mocha-client-tests/addon/tests/mocha-run.js

```diff
@@ -0,0 +1,3 @@
+mocha.checkLeaks();
+mocha.globals(['jQuery','AppView', 'ExtensionOptions', 'ExtensionView', 'WebView']);
```

Comment:
where are these globals used? These don't look like they are relevant to the example